centreon-open-tickets/www/modules/centreon-open-tickets/providers/EasyVistaRest/EasyVistaRestProvider.class.php

```
@@ -57,12 +57,89 @@
+
+    /*
+    * check if the close option is enabled, if so, try to close every selected ticket
+    *
+    * @param {array} $tickets
+    *
+    * @return void
+    */
+    public function closeTicket(&$tickets): void
+    {
+        if ($this->doCloseTicket()) {
+            foreach ($tickets as $k => $v) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$v'. (Unused Code Rules)

(UnusedLocalVariable)